src/api/spec/models/project_spec.rb

```diff
@@ -585,4 +585,17 @@ def reset_project_in_backend
       expect(project_release.packages.where(name: 'my_release_target')).to exist
     end
   end
+
+  describe 'foo' do
```

Comment:
^Looks like a leftover:laughing:

Reply:
I was like Michael Douglas @ Falling down :P 
couldnt test because of the issue with createrepo_c, then merged an approved PR and the whole docker rake tasks are gone and I couldnt test my code anymore.. I was like https://www.youtube.com/watch?v=zJs9p-VNORw :P